01-getting-started/003-ecr-setup.md

```diff
@@ -10,8 +10,41 @@ This guide will guide you through the creation of an ECR (Elastic Container Regi
 
 AWS resources are provisioned through the [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments/) repository, per environment. Your application might be using multiple environments, however, you only need one image repository so we will be working in only one of the environments (it doesn't matter which one).
 
-## Write the terraform configuration
-In your local clone of the environments repository, `cd` to your environment's path and create a new directory called `resources`. Inside `resources`, create a new file called `ecr.tf` with the following contents:
+### Creating the registry
+
+1\. In order to create the ECR Docker registry, git clone the repo and create a new branch.
```

Comment:
I think we should change this a little to say we want people to make a fork of the repo and then do all the rest on their fork. This means we don't have to add each person individually to the `cloud-platform-environments` repo. I'll ask for thoughts on that in the chat.

Reply:
Okay based on the chat we have had we'll keep it this way and add teams to the repo up front. This will:

1. Reduce the number of long lived forks that are around which we may want to get rid of e.g. when someone leaves the org
2. Make sure most users have access to the repo from day 1

So no changes here.